node-module-system/add.ts

```diff
@@ -0,0 +1,3 @@
+export function addTwoNumbers(number1: number, number2: number): number {
```

Comment:
"two numbers" is assumed, so it's unnecessary and a bit verbose to include that in the name of the function.